README.md

```diff
@@ -126,6 +126,18 @@ As the methods are pure, and to avoid re-asking the user for credentials everyti
 }
 ```
 
+#### purchases
+```javascript
+{
+  id, // a string that uniquely represents this activity
+  datetime, // a javascript Date object that represents the start of the activity
+  label, // a string that represents the transaction
+  merchantDisplayName, // a string that represents the merchant where the purchase was made
```

Comment:
We don't use `merchantDisplayName` atm, but it makes sense to have.  Can you add "(optional)"?